docs/ingress.md

```diff
@@ -0,0 +1,84 @@
+# Add route to Kubeless function
+
+Kubeless leverages [Kubernetes ingress](https://kubernetes.io/docs/concepts/services-networking/ingress/) to provide routing for function. By default, a deployed function will be matched to a Kubernetes service with service type is ClusterIP that means it is not exposed publicly. We provides `kubeless ingress` command to make the function published. This guide give you a quick sample on how to do it.
+
+## Ingress controller
+
+In order to create route for function in Kubeless, you must have an Ingress controller running. There are several options to deploy it. You can deploy it manually via the manifest we provide. If you are on GKE, you can try [this](https://github.com/kubernetes/ingress/tree/master/controllers/gce). Minikube also provide an addon for ingress, enable it with this command `minikube addons enable ingress`
```

Comment:
Please, add a link to our manifest. Also it would be good to mention that if you have the minikube addon enabled, you won't be able to install the manifest